src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartGalaxyController.java

```diff
@@ -12,27 +13,26 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import ca.corefacility.bioinformatics.irida.model.project.Project;
 import ca.corefacility.bioinformatics.irida.model.sample.Sample;
 import ca.corefacility.bioinformatics.irida.ria.config.GalaxySessionInterceptor;
-import ca.corefacility.bioinformatics.irida.ria.web.cart.components.Cart;
-import ca.corefacility.bioinformatics.irida.ria.web.cart.dto.CartSample;
 import ca.corefacility.bioinformatics.irida.ria.web.cart.dto.GalaxyExportSample;
+import ca.corefacility.bioinformatics.irida.ria.web.services.UICartService;
 import ca.corefacility.bioinformatics.irida.service.sample.SampleService;
 
 /**
  * Controller to handle all ajax requests made to the cart that have to do with Galaxy.
  */
 @RestController
-@Scope("session")
 @RequestMapping("/ajax/galaxy-export")
 public class CartGalaxyController {
-	private SampleService sampleService;
-	private Cart cart;
+	private final SampleService sampleService;
```

Comment:
Remove the `final` from the variables

Reply:
Updated in 0019e2334d708eab552b72a2f9e3acea265c71c3